pig/src/main/resources/interpreter-setting.json

```diff
@@ -8,19 +8,25 @@
         "envName": null,
         "propertyName": "zeppelin.pig.execType",
         "defaultValue": "mapreduce",
-        "description": "local | mapreduce | tez"
+        "description": "local | mapreduce | tez_lcoal | tez | spark_local | spark"
       },
       "zeppelin.pig.includeJobStats": {
         "envName": null,
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
       },
-      "zeppelin.pig.execType": {
+      "SPARK_MASTER": {
         "envName": "SPARK_MASTER",
         "propertyName": "SPARK_MASTER",
         "defaultValue": "local",
         "description": "local | yarn-client"
+      },
+      "SPARK_JAR": {
+        "envName": "SPARK_JAR",
+        "propertyName": "SPARK_JAR",
+        "defaultValue": "",
+        "description": "spark assembly jar uploaded in hdfs"
```

Comment:
Hmm, right. Seems this is a bug of pig. Pig says that this should be a file in hdfs, but I verified that local jar also works. https://github.com/apache/pig/blob/trunk/bin/pig#L408